special-pages/pages/new-tab/app/next-steps/NextStepsCard.js

```diff
@@ -0,0 +1,27 @@
+import {h} from "preact"
+import cn from "classnames"
+import styles from "./NextSteps.module.css"
+import { Cross } from '../components/Icons.js'
+import {variants} from "./nextsteps.data"
+
+export function NextStepsCard({type, dismiss}) {
+    const message = variants[type]
```

Comment:
We'll need to translate the text here, so you can make this a function call and pass in the `t` param from `useTypedTranslations()`

```js
const { t } = useTypedTranslations();
const message = variants[type]?.(t);
```

and then inside the data file, make each item accept `t` as a param.

The translations need to be added to `special-pages/pages/new-tab/src/locales/en/newtab.json` for now - you can add them to the bottom of the file and use the namespacing for separation, like

```json
{
  "...": "...",
  "nextSteps_abc": {...} 
}
```